template.tpl

```diff
@@ -318,6 +318,44 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "additionalInformation",
+    "groupStyle": "ZIPPY_CLOSED",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "products",
+        "displayName": "Product Information",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Product ID",
+            "name": "id",
+            "type": "TEXT",
+            "valueHint": "The item\u0027s ID in the catalog"
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Name",
+            "name": "name",
+            "type": "TEXT",
+            "valueHint": ""
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Category",
+            "name": "category",
+            "type": "TEXT",
+            "valueHint": "e.g. IAB content taxonomy",
```

Comment:
```suggestion
            "valueHint": "e.g., IAB content taxonomy",
```
Slight grammar typo.

My preference would be for writing out the words to make it as broadly understandable as possible, but YMMV.